app/src/main/java/com/duycuong/weather/data/source/local/SharePreferences.java

```diff
@@ -0,0 +1,42 @@
+package com.duycuong.weather.data.source.local;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+
+/**
+ * Created by DuyCương on 22/02/2018.
+ */
+
+public class SharePreferences {
+    private static final String UNIT = "unit";
+    private static final String TEMPERATURE = "temperature";

```

Comment:
2.10 String constants, naming and values
Many elements of the Android such as SharedPreferences, Bundle or Intent use a key-value pair.
https://github.com/framgia/coding-standards/blob/master/eng/android/codingstyleguide.md